birdhouse/config/finch/wps.cfg.template

```diff
@@ -2,6 +2,9 @@
 outputurl = https://${PAVICS_FQDN_PUBLIC}/wpsoutputs
 outputpath = /data/wpsoutputs
 
+# default 3mb, fix "Broken pipe" between the proxy and the wps service
+maxrequestsize = 100mb
```

Comment:
Any possible side effect of this change?

Reply:
Not sure, I am guessing if someone upload such a large `.geojson` file it will be a heavier calculation... so performance would be a side-effect I guess.  But otherwise, @huard said it's a reasonable value for production server.